_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/package.json

```diff
@@ -8,26 +8,21 @@
   },
   "scripts": {
     "start": "npm run scripts:dev",
-    "build": "npm run del && npm run styles && npm run scripts",
+    "build": "npm run styles && npm run scripts",
     "serve": "spandx -c spandx.config.dev.js",
     "test": "karma start",
-    "del": "del 'node_modules/@rhd' && del dist/js",
-    "del:styles": "del \"dist/css**\" && del 'node_modules/@rhdstyles'",
-    "fix:fa": "rename -n \"node_modules/@fortawesome/!(fontawesome-common-types)/index.d.ts\" index.es.d.ts",
-    "scripts:dev": "npm run scripts:old && npm run scripts:microsites && npm run scripts:new && npm run scripts:tsdev && npm run scripts:copy",
-    "scripts": "npm run scripts:old && npm run scripts:microsites && npm run scripts:new && npm run scripts:ts && npm run scripts:copy",
-    "scripts:copy": "copyfiles -u 2 \"node_modules/@patternfly/**/*\" \"dist/js/@patternfly\" && copyfiles -u 2 \"node_modules/@fortawesome/**/*\" \"dist/js/@fortawesome/\"",
-    "scripts:new": "uglifyjs-folder src/scripts/js/rhd -c -o dist/js/@rhd/rhd.min.js --source-map",
-    "scripts:old": "uglifyjs-folder src/scripts/js/2018 -c -o dist/js/@rhd/rhd.old.min.js --source-map",
-    "scripts:microsites": "uglifyjs-folder src/scripts/js/microsites -c -o dist/js/@rhd/rhd.microsites.min.js --source-map",
-    "scripts:ts": "tsc && copyfiles -u 4 \"dist/js/src/scripts/@rhd/**/*\" dist/js/node_modules/ && rollup -c rollup.config.js",
+    "scripts:dev": "npm run scripts:old && npm run scripts:new && npm run scripts:tsdev",
+    "scripts": "npm run scripts:old && npm run scripts:new && npm run scripts:ts && npm run scripts:copy",
+    "scripts:copy": "copyfiles \"node_modules/@patternfly/pfelement/pfelement.umd.min.js\" \"dist/js/patternfly/\" && copyfiles \"node_modules/@patternfly/pfelement/pfelement.umd.min.js.map\" \"dist/js/patternfly/\" && copyfiles \"node_modules/@patternfly/pfe-datetime/pfe-datetime.umd.min.js.map\" \"dist/js/patternfly/\" && copyfiles \"node_modules/@patternfly/pfe-datetime/pfe-datetime.umd.min.js.map\" \"dist/js/patternfly/\"",
```

Comment:
There is a not found for /themes/custom/rhdp2/rhd-frontend/dist/js/patternfly/pfelement/pfelement.umd.min.js?q5cr95
which appears to cause a few js console errors on the homepage.